packages/bpk-component-barchart/src/BpkBarchartBar.js

```diff
@@ -51,35 +51,53 @@ const BpkBarchartBar = (props) => {
     onClick,
     outlier,
     selected,
+    padding,
```

Comment:
Fair enough. So as the API between bars and bar is private, I guess it's ok to postpone documenting it for now?